contrib/intel_env_vars_list.txt

```diff
@@ -19,3 +19,5 @@ CUFFDIFF_SCRIPT_PATH
 PATH
 NSLOTS
 PYTHON_EGG_CACHE
+LABKEY_USERNAME
```

Comment:
I think these entries should be entered in the same order as contrib/intel_env.sh or vice versa.